### ISSUE TITLE
New media types not available after upgrade to 8.14

### ISSUE THREAD

**Issue by dawoe** [umbraco/Umbraco-CMS#10380]
### Which Umbraco version are you using?

8.14

### Bug summary

After an upgrade from 8.13 to 8.14 the new media types mentioned here are not available : https://our.umbraco.com/Documentation/Fundamentals/Data/Creating-Media/

I also checked the version specific upgrade instructions but don't see anything metioned on how to get these new media types.

On a fresh install they are available

### Specifics

Upgrade was done on 8.13 site using the default starter kit to 8.14

### Steps to reproduce

Setup a website using Umbraco 8.13 or earlier with the defautl starter kit.

Upgrade the website to 8.14

#### Expected

The new media types are available.

#### Actual

The new media types are not available.

**Comment by nul800sebastiaan** [comment]
Correct, on upgraded installs you can create them, but we should document it. I don't think it should be in the version-specific docs though, that's only to make sure that you can upgrade without anything breaking. This is just an additional feature, if you don't create new mediatypes then everything will and should work exactly like it did before. 

But good to add to the linked documentation I would say, I'll move to the Docs repo.
